knimin/lib/data_access.py

```diff
@@ -2425,3 +2553,38 @@ def _revert_ready(self, barcodes):
                  SET results_ready = NULL
                  WHERE barcode IN %s"""
         self._con.execute(sql, [tuple(barcodes)])
+
+    def _add_new_samples(self, sample_ids):
```

Comment:
Call this "create_sample" and add tests for it, it will be a function that we will be using at some point (not only for tests) so since you've already written it we can make it available